CONTRIBUTING.md

```diff
@@ -122,9 +122,8 @@ should run:
 
 ```shell
 cd path/to/repo/checkout
-python -m venv venv
-source venv/bin/activate
-pip install -r test-requirements.txt  # installs e.g. pytest
+uv sync
```

Comment:
Are there any dependencies we would need to mention in order to provide uv command in the first place?

Reply:
Other than `uv` itself, I don't believe so.